LICENSE

```diff
@@ -0,0 +1,671 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
You might want to check in with @nedbat or @jbarciauskas about this LICENSE. I'm not sure if it is our policy to mark our authored xBlocks as AGPL or Apache...